assignment-section-05/generate_test_data.py

```diff
@@ -0,0 +1,26 @@
+from pathlib import Path
```

Comment:
this file is new respect to the house price, isn't it?

I doubt very much that students will imagine doing anything like what is going on here. I would not add extra files.